plot/initial-templates.json

```diff
@@ -0,0 +1,163 @@
+{"mark": "bar", "encoding": {"x": "temporal", "y": "count", "size": "nominal"}},
+{"mark": "bar", "encoding": {"x": "temporal", "y": "quantitative_mean"}},
+{"mark": "bar", "encoding": {"x": "temporal", "y": "quantitative_mean"}},
+{"mark": "box-plot", "encoding": {"x": "ordinal", "y": "quantitative"}},
+{"mark": "box-plot", "encoding": {"x": "quantitative", "y": "ordinal"}},
+{"mark": "box-plot", "encoding": {"x": "quantitative"}},
+{"mark": "box-plot", "encoding": {"y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "column": "nominal"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "column": "nominal"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "opacity": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "count"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative", "size": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "quantitative", "y": "quantitative"}},
+{"mark": "circle", "encoding": {"x": "temporal", "y": "temporal", "size": "quantitative_sum"}},
+{"mark": "line", "encoding": {"x": "ordinal", "y": "quantitative_median", "color": "nominal"}},
+{"mark": "line", "encoding": {"x": "quantitative", "y": "quantitative", "order": "temporal"}},
+{"mark": "line", "encoding": {"x": "temporal", "y": "quantitative", "color": "nominal"}},
+{"mark": "line", "encoding": {"x": "temporal", "y": "quantitative", "detail": "nominal"}},
+{"mark": "line", "encoding": {"x": "temporal", "y": "quantitative"}},
```

Comment:
lots of duplicates here, skew distribution?

Reply:
This is intended. Combinations that appear in more spec files might be more common ones, so I want to sample them more frequently.